src/pages/home/report/ReportActionsList.js

```diff
@@ -162,21 +158,19 @@ function ReportActionsList(props) {
     // Native mobile does not render updates flatlist the changes even though component did update called.
     // To notify there something changes we can use extraData prop to flatlist
     const extraData = [props.isSmallScreenWidth ? props.newMarkerReportActionID : undefined, ReportUtils.isArchivedRoom(props.report)];
-    const shouldShowReportRecipientLocalTime = ReportUtils.canShowReportRecipientLocalTime(props.personalDetailsList, props.report, props.currentUserPersonalDetails.accountID);
-
     const errors = lodashGet(props.report, 'errorFields.addWorkspaceRoom') || lodashGet(props.report, 'errorFields.createChat');
-    const isArchivedRoom = ReportUtils.isArchivedRoom(props.report);
     const hideComposer = ReportUtils.shouldHideComposer(props.report, errors);
-    const shouldOmitBottomSpace = hideComposer || isArchivedRoom;
+    const shouldShowReportRecipientLocalTime =
+        ReportUtils.canShowReportRecipientLocalTime(props.personalDetails, props.report, props.currentUserPersonalDetails.accountID) && !props.isComposerFullSize;
 
     return (
-        <Animated.View style={[animatedStyles, styles.flex1]}>
+        <Animated.View style={[animatedStyles, styles.flex1, !shouldShowReportRecipientLocalTime && !hideComposer && styles.pb4]}>
```

Comment:
I think it was a good idea to completely remove `shouldShowReportRecipientLocalTime` in this component and instead add in `ReportActionCompose`. Any issue you faced to fix reported regression?

Reply:
I know but what I thought was to add margin in parent container View instead of child [View](https://github.com/Expensify/App/blob/811a83e28b98a7c348081fc0dbfe1f658c57f1d0/src/pages/home/report/ReportActionCompose.js#L1028).
https://github.com/Expensify/App/blob/811a83e28b98a7c348081fc0dbfe1f658c57f1d0/src/pages/home/report/ReportActionCompose.js#L1011

Anyway no problem. Current solution is more safe.